BioSQL/BioSeq.py

```diff
@@ -130,10 +132,20 @@ def toseq(self):
         return Seq(str(self), self.alphabet)
 
     def __add__(self, other):
+        """Add another sequence or string to this sequence.
+
+        The sequence is first converted to a Bio.Seq object before the addition.
+        The returned object is a Bio.Seq, not a BioSQL.DBSeq
+        """
         # Let the Seq object deal with the alphabet issues etc
         return self.toseq() + other
 
     def __radd__(self, other):
+        """Add another sequence or string to the left.
+
+        The sequence is first converted to a Bio.Seq object before the addition.
+        The returned object is a Bio.Seq, not a BioSQL.DBSeq
```

Comment:
See my general comment about what to call the ``Seq`` object. Likewise this is not the ``BioSQL.DBSeq`` object. In full it is the ``BioSQL.BioSeq.DBSeq`` object, so I would recommend just calling it the ``DBSeq`` object.